index.js

```diff
@@ -0,0 +1,52 @@
+const chromeLauncher = require('chrome-launcher');
```

Comment:
Looks awesome! Will this be runnable in gh-pages?

Reply:
Done in #14 ! Please have a look.

Reply:
@jywarren Yes. The gh-pages is served from the browserified `orb.core.min.js` file generated from the src folder.